api/package-lock.json

```diff
@@ -15,11 +15,13 @@
         "bowser": "^2.11.0",
         "buffer-shims": "^1.0.0",
         "compression": "^1.7.4",
+        "cron-validator": "^1.3.1",
         "express": "^4.18.2",
         "google-libphonenumber": "^3.2.33",
         "gsm": "^0.1.4",
         "helmet": "^3.23.3",
         "http-proxy": "^1.18.1",
+        "later": "^1.2.0",
```

Comment:
These have been added to the package lock but not the package? Should this file be reverted?